### ISSUE TITLE
Issue with conditional-gan for tensorflow version

### ISSUE THREAD

**Issue by janker5** [wiseodd/generative-models#72]
Hey guys, I'm using the conditional-gan code for tensorflow version to generate garment drawings, the training images is RGB png file, labels are vectors like this [1,2,3,4,...,2]，I just changed the feeding training data from mnist to my data, but the generated images are shapeless, I don't know why, someone helps me? thanks!!
The attached is the modified code and images generated.

[code+generated images.zip](https://github.com/wiseodd/generative-models/files/2897643/code%2Bgenerated.images.zip)

**Comment by janker5** [comment]
BTW, the training loss for the generator is always very large(>20), while for the discriminator is very small(< 0.001), is there something wrong with the training system?
